ts/a11y/explorer/Highlighter.ts

```diff
@@ -378,16 +466,96 @@ class SvgHighlighter extends AbstractHighlighter {
    * @override
    */
   public unhighlightNode(info: Highlight) {
-    const previous = info.node.previousSibling as HTMLElement;
-    if (previous && previous.hasAttribute('sre-highlighter-added')) {
+    const node = info.node;
+    const previous = node.previousSibling as HTMLElement;
+    if (node.hasAttribute('data-sre-highlighter-bbox')) {
+      node.remove();
+      return;
+    }
+    node.removeAttribute('data-mjx-enclosed');
+    if (previous && previous.hasAttribute('data-sre-highlighter-added')) {
       info.foreground
-        ? info.node.setAttribute('fill', info.foreground)
-        : info.node.removeAttribute('fill');
-      info.node.parentNode.removeChild(previous);
+        ? node.setAttribute('fill', info.foreground)
```

Comment:
Here we are resetting only on node to the original foreground color. However, that should be all the nodes in the enclosed element.